discord/guild.go

```diff
@@ -53,23 +141,30 @@ func (guild *GuildState) muteAllTrackedMembers(dg *discordgo.Session, mute bool,
 				}
 			}
 			buf.WriteString(fmt.Sprintf("Username: %s, Nickname: %s, ID: %s", v.user.userName, v.user.nick, user))
-			//buf.WriteString(v.User.Username)
-			//if v.Nick != "" {
-			//buf.WriteString(fmt.Sprintf(" (%s)", v.Nick))
-			//}
 			log.Println(buf.String())
 			if !skipExec {
 				err := guildMemberMute(dg, guild.ID, user, mute)
 				if err != nil {
 					log.Println(err)
 				}
-				log.Printf("Sleeping for %dms between mutes to avoid being rate-limited by Discord\n", guild.delays.DiscordMuteDelayOffsetMs)
+				log.Printf("Sleeping for %dms between actions to avoid being rate-limited by Discord\n", guild.delays.DiscordMuteDelayOffsetMs)
 				time.Sleep(time.Duration(guild.delays.DiscordMuteDelayOffsetMs) * time.Millisecond)
 			}
 		}
 	}
 	guild.voiceStatusCacheLock.RUnlock()
 }
+
+func guildMemberMove(session *discordgo.Session, guildID string, userID string, channelID *string) (err error) {
```

Comment:
took this from the discordgo source, since i couldn't access the library's method for this 🤔

Reply:
Yep, had the exact same problem with the Mute command... Really not sure why some methods are in the docs, but NOT in the source... Couldn't find a good reason for the discrepancies...